packages/UI/src/components/Checkbox/Checkbox.mdx

```diff
@@ -1,6 +1,6 @@
-import { Canvas, Meta, Controls } from '@storybook/blocks';
-
-import * as CheckboxStories from './Checkbox.stories';
+import { Canvas, Meta, Controls, Stories } from '@storybook/blocks';
+import * as CheckboxStories from './Checkbox.stories.tsx';
+import Checkbox, { checkboxStyles } from './Checkbox.tsx';
```

Comment:
it's preferred to keep the imports without the file extension. 
in this case it's better to have 
`import * as CheckboxStories from './Checkbox.stories';`
instead of 
`import * as CheckboxStories from './Checkbox.stories.tsx';`
as an example.